<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDb2NewConnectionBase</class>
 <widget class="QDialog" name="QgsDb2NewConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New DB2 Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QgsMessageBar" name="bar" native="true">
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtPort"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtHost"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DB2ServiceLabel">
        <property name="text">
         <string>Service/DSN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtService"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="DB2ServiceLabel_2">
        <property name="text">
         <string>Driver</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="DB2HostLabel">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="database">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtDatabase"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="DB2port">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="DB2ServiceLabel_3">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDriver"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mAuthGroupBox">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QgsAuthSettingsWidget" name="mAuthSettings" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>&amp;Test Connection</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>auth/qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QWidget</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtService</tabstop>
  <tabstop>txtDriver</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtDatabase</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>released()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>on_btnConnect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
